README.md

```diff
@@ -90,14 +85,9 @@ and health-checking mechanisms.
 
 ### Build and install `hare` rpm from source.
 **NOTE: If you have built Motr and HARE from source you will not need to generate RPM packages below**
-* Install Motr.
-
-  * .. from RPMS
-    ```sh
-    sudo yum -y install cortx-motr cortx-motr-devel
-    ```
+* Build `Motr` RPMs.
```

Comment:
I think this is the only place where Motr is highlighted as `Motr` (code block). Let's keep it consistent with other entries in this text.

Reply:
Resolved these comments as part of #1610